pkg/server/sandbox_run.go

```diff
@@ -639,3 +641,15 @@ func (c *criService) getSandboxRuntime(config *runtime.PodSandboxConfig, runtime
 	}
 	return handler, nil
 }
+
+func logDebugCNIResult(sandboxID string, result *cni.CNIResult) {
+	if logrus.GetLevel() != logrus.DebugLevel {
```

Comment:
We also have a trace level that is DebugLevel + 1. See our use of "github.com/containerd/containerd/log" throughout. So let's go ahead and import that package here and do a switch or just make it >= DebugLevel. Then test it with trace on..

Reply:
I hope I addressed what described.
If the iota Level is lower than 5 (debug) we don't log (e.g: 4 == info).

Reply:
yes I meant continue if >= I forgot you were returning early so yeah return if < debug 
:-) heh